config/prometheus/monitor.yaml

```diff
@@ -11,6 +11,10 @@ spec:
   endpoints:
     - path: /metrics
       port: https
+      scheme: https
+      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      tlsConfig:
+        insecureSkipVerify: true
```

Comment:
Is this also part of the migration? Or did you have to set this manually?

Reply:
Yes and yes. It was suggested by migration guide and had to be done manually. https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.6.1/#gov2-gov3-ansiblev1-helmv1-add-scheme-token-and-tls-config-to-the-prometheus-servicemonitor-metrics-endpoint

Reply:
I'm a bit uneasy about this, as we have all the pieces there to make a proper secure connection. I'm opening an issue to track this and merge as is, as it's certainly an improvement to what we have today (no TLS at all).

Reply:
Created #308